### PR TITLE
Add tests for nfs* frames v3

### PR DESCRIPTION
**Pull request by ghostinthehive**
Previous PR #739 
Changes from Previous PR:

- Add `content` modifier keywords for needed rules
 
Goes with https://github.com/OISF/suricata/pull/7043

### CODE REVIEW

File: tests/nfs2-frames/test.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+requires:
+  min-version: 7.0
+  files:
+      - rust/src/nfs/nfs2.rs
```
Comment (jufajardini) [inline tests/nfs2-frames/test.yaml:4]: I think we don't need to use that any longer, these days for Rust is not optional.

**Review by jufajardini — CHANGES_REQUESTED** [review]
Looking good to me! Just a minor thing from my end, see inline comment :)

**Comment by inashivb** [comment]
Another nit: I think we can drop the `S-V` in the commit message.

**Comment by ghostinthehive** [comment]
Followed by #754

**Pull request closed without merging**